modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -770,7 +770,10 @@ GAPI_EXPORTS GMat mulC(const GScalar& multiplier, const GMat& src, int ddepth =
 The function divides one matrix by another:
 \f[\texttt{dst(I) = saturate(src1(I)*scale/src2(I))}\f]
 
-When src2(I) is zero, dst(I) will also be zero. Different channels of
+For integer types when src2(I) is zero, dst(I) will also be zero.
+Floating point case returns Inf/NaN (according to IEEE).
```

Comment:
Does this PR resolve #13147 ?

Reply:
Unfortunately, I haven't heard about this issue earlier. So I didn't check resolving this issue. I can say with confidence that during testing this snippet with `DivPerfTestFluid`  test behavior of Div fluid's kernel is fully consistent with the behavior of `cv::divide()` operation. The exception is the behavior is caused with known issue in OpenCV (dtype== CV_16S && type != dtype)

Reply:
Since all CI checks successfully passed, I can assume that mentioned issue in PR #13147 was successfully resolved.